velox/common/caching/AsyncDataCache.cpp

```diff
@@ -177,6 +177,7 @@ CachePin CacheShard::findOrCreate(
           found->setPrefetch(false);
         } else {
           ++numHit_;
+          hitBytes_ += found->size();
```

Comment:
Why we not count `hitBytes_` if found is prefetch? Data has already loading in memory right? and we actually return the prefetched entry in this branch end.